core/autoscaling.go

```diff
@@ -334,6 +335,40 @@ func (a *autoScalingGroup) replaceOnDemandInstanceWithSpot(
 	return a.detachAndTerminateOnDemandInstance(odInst.InstanceId)
 }
 
+func (a *autoScalingGroup) getInstanceTypeByTagInASG() (string, error) {
```

Comment:
Sorry for wasting your time, I missed it in the previous review.

I was referring to this: https://docs.aws.amazon.com/sdk-for-go/api/service/autoscaling/#Group, notice that it has a Tags field which should be already populated.

The other functionality using configurations set on the group already takes the tags from there, have a look at https://github.com/cristim/autospotting/blob/master/core/autoscaling.go#L76 and https://github.com/cristim/autospotting/blob/master/core/autoscaling.go#L756